### ISSUE TITLE
Email: Header images are inside a layout table that is spoken as "table start, table end" to VoiceOver.

### ISSUE THREAD

**Issue by JohnHewittCriterion** [nciinc/fs-permit-platform#618]
## Notes
The header images are inside a layout table and the screen reader is speaking table start and end information which is purely decorative so the table semantics should be hidden.
 
Add role=presentation to the <table> tag to hide the data table semantics from the screen reader user. Or just use DIV and CSS rather than <table> layout. 

![image.png](https://images.zenhubusercontent.com/5913513251aa2c30577de73e/e682fe7c-cff5-49bf-a42d-9bff6d19b302)

Linkage: CTF-101
WCAG: 1.3.1 Info and Relationship 

## Acceptance criteria
- [ ] table semantics hidden to screen reader

## Tasks
- [x] add role="presentation" can't use div in email html (will)

## Definition of Done
- [x] Code unit / e2e tested and comply with QWASP
- [x] No new security vulnerabilities
- [x] Internal team code review
- [x] Documentation / readme.md / CircleCI updated
- [x] Docker updated
- [x] Compare finished design with mockup
- [ ] Usability tested

**Comment by sadlerw** [comment]
OK so yahoo is taking the role out of this for me.  I'm not sure if it's working or just yahoo :-(
